gfx_scene.h

```diff
@@ -1602,7 +1602,7 @@ class GfxSceneInternal
             }
         }
         std::set<uint64_t> unparented_nodes;
-        std::map<int32_t, uint64_t> animated_nodes;
+        std::map<int32_t, std::pair<size_t, uint64_t>> animated_nodes;
```

Comment:
`animated_nodes` is shared between all animations but each animated node only belongs to one animation. So we store the animation index with the animated node handle.